src/widget.tsx

```diff
@@ -285,25 +285,26 @@ export class LuxWidgetView extends DOMWidgetView {
       render() {
         var buttonsEnabled = Object.keys(this.state._selectedVisIdxs).length > 0;
         var intentEnabled = Object.keys(this.state._selectedVisIdxs).length == 1 && Object.values(this.state._selectedVisIdxs)[0].length == 1;
-        
         if (this.state.recommendations.length == 0) {
           return (<div id="oneViewWidgetContainer" style={{ flexDirection: 'column' }}>
-                  {/* {attributeShelf}
-                  {filterShelf} */}
-                  <div style={{ display: 'flex', flexDirection: 'row' }}>
-                    <CurrentVisComponent intent={this.state.intent} currentVisSpec={this.state.currentVis} numRecommendations={0}
-                    onChange={this.handleCurrentVisSelect}/>
-                  </div>
-                  <ButtonsBroker buttonsEnabled={buttonsEnabled}
+                    <div style={{ display: 'flex', flexDirection: 'row' }}>
+                      <CurrentVisComponent intent={this.state.intent} currentVisSpec={this.state.currentVis} numRecommendations={this.state.recommendations.length}
+                      onChange={this.handleCurrentVisSelect}/>
+                    </div>
+                    <ButtonsBroker buttonsEnabled={buttonsEnabled}
                                      deleteSelection={this.deleteSelection}
                                      exportSelection={this.exportSelection}
                                      setIntent={this.setIntent}
                                      closeExportInfo={this.closeExportInfo}
                                      tabItems={this.state.tabItems}
                                      showAlert={this.state.showAlert}
                                      intentEnabled={intentEnabled}
-                                     />               
-                </div>);
+                                     />
+                      <div id="no-recs-footer" style={{display:"flex"}}>
+                        <div id="no-recs" className = "fa fa-exclamation-triangle"></div>
```

Comment:
Do we want to remove the bullet for the list item here? In this case, it is only possible to have a single bullet item right?

![image](https://user-images.githubusercontent.com/5554675/109386735-b9a9b080-7937-11eb-8d3b-0fe531c24333.png)

Reply:
Fixed in next push!